src/main/java/edu/kis/powp/jobs2d/command/FigureFactory.java

```diff
@@ -0,0 +1,50 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FigureFactory {
+
+    public static ComplexCommand getSquare(DriverManager driverManager) {
```

Comment:
DriverCommand return type would be sufficient
Passing a manager instead of a driver increases dependencies.